cg/apps/demultiplex/sample_sheet/api.py

```diff
@@ -80,6 +94,78 @@ def validate_sample_sheet(
             file_path=sample_sheet_path, bcl_converter=bcl_converter
         )
 
+    @staticmethod
+    def _is_sample_sheet_from_flow_cell_translatable(flow_cell: FlowCellDirectoryData) -> bool:
+        """
+        Determine if the sample sheet from the flow cell directory is translatable to BCLConvert.
+        """
+        try:
+            flow_cell.run_parameters_path.exists()
+        except FlowCellError:
+            LOG.error(f"Run parameters file for flow cell {flow_cell.full_name} does not exist")
+            return False
```

Comment:
Exists does not return an error it returns a bool I think this is what you want, maybe it can be incorporated into an `necessary_files_exist()` method together with the one below
```suggestion
        if not flow_cell.run_parameters_path.exists()
            LOG.error(f"Run parameters file for flow cell {flow_cell.full_name} does not exist")
            return False
```

Reply:
Unfortunately `flow_cell.run_parameters_path` raises an error if the run parameters does not exists, so the `try-except` clause is needed here. I think that maybe this could not be the best behaviour for this function but I think that changing this behaviour would be out of the scope of this PR. 
I could however remove the `.exists()` and give it a bit more clarity to this part, what do you think? @Vince-janv

Reply:
hmm. Does it make sense thet the property `run_parameters_path` raises an error? Maybe it can just return `None` if there is not file. That would make more sense to me. What do you think?

Reply:
It makes sense that it returns None instead of error. I would have to see if fixing that would extend tremendously the scope of this PR, as this attribute is used in many places

Reply:
will be solved in another PR, described in #3083